specs/endpoint-security.spec.yml

```diff
@@ -14,6 +14,8 @@ inputs:
       - UPGRADE
     runtime:
       preventions:
+        - condition: ${runtime.arch} != ${runtime.native_arch}
```

Comment:
Should this be done for MacOS as well, probably the platform most likely for this to occur?

https://github.com/elastic/elastic-agent/blob/434986ed9fa6fbed6257557a4c6773d519742f66/specs/endpoint-security.spec.yml#L51-L63

I swore I saw this change before and I did, but it was only for Windows https://github.com/elastic/elastic-agent/pull/4396/

Reply:
Perhaps you're right. For macOS we build a fat binary, containing executable for x86_64 and arm64, but I remember that the execution model follows the architecture of the parent process so x86_64 Agent will execute the fat binary as x64_64. I'll test it. Thanks